docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,19 +24,15 @@ Fixes # (issue number, *if applicable*)
 
 Closes # (issue number, *if applicable*)
 
-Polkadot companion: (*if applicable*)
-
-Cumulus companion: (*if applicable*)
-
 # Checklist
 
 - [ ] My PR includes a detailed description as outlined in the "Description" section above
 - [ ] My PR follows the [labeling requirements](CONTRIBUTING.md#Process) of this project (at minimum one label for `T`
   required)
 - [ ] I have made corresponding changes to the documentation (if applicable)
 - [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
-- [ ] If this PR alters any external APIs or interfaces used by Polkadot, the corresponding Polkadot PR is ready as well
-  as the corresponding Cumulus PR (optional)
+- [ ] If this PR alters any external APIs or interfaces used by the Polkadot runtimes,
+     the corresponding `polkadot-fellows/runtimes` PR is ready as well (optional)
```

Comment:
```suggestion
```

No companions! This will require proper docs on how to migrate code etc. In the future this will be done with the PRDOC stuff.